stdlib/ballerina-database/src/main/ballerina/h2/Package.md

```diff
@@ -0,0 +1,27 @@
+## Package overview
+
+This package provides the functionality required to access and manipulate data stored in h2 database. 
+
+### Endpoint 
+
+To access a database, you must first create an `endpoint`, which is a virtual representation of the physical endpoint of the h2 database that you are trying to connect to. Create an endpoint of the H2 client type (i.e., `h2:Client`) and provide the necessary connection parameters. This will create a pool of connections to the given h2 database. A sample for creating an endpoint with h2 client can be found below. 
```

Comment:
Change "h2" to "H2" throughout this doc (except when it's part of a package name, function, etc.)

Reply:
Change "A sample for creating an endpoint with h2 client" to "A sample for creating an endpoint with an H2 client"